### PR TITLE
document who is expected to attend membership meetings

### PR DESCRIPTION
**Pull request by crenshaw-dev**
The Argoproj approvers and leads have selected several contributors to be "scoped" approvers, i.e. folks who are able to merge some (but not all) types of pull requests in their subproject.

To keep membership meetings focused and efficient, we'd like to clarify the governance documentation to specify that the meetings are attended by "full approvers," i.e. contributors who have been granted access to merge _any_ PR in their subproject.

We think this limitation will be helpful as we try out the new "scoped approvers" system.

Needed to pass:
* Votes by 30% of maintainers (at least 13): ✔️   currently 17 votes

### CODE REVIEW

File: community/membership.md [hunk 1]
```diff
@@ -268,6 +268,7 @@ before being able to contribute effectively.
 
 A change in membership role may be approved by consensus of the Argo Project leads and approvers. By convention, role changes are discussed in a meeting with Argo Project approvers before being finalized. Role changes are generally reviewed on a quarterly basis.
 
+The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings efficient by focusing the discussion among those who are best-positioned to evaluate new memberships and promotions.
```
Comment (edlee2121) [inline community/membership.md:271]:
```suggestion
The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings at a reasonable size.
```

Comment (crenshaw-dev) [reply community/membership.md]: Short and sweet. Thanks!

**Review by terrytangyuan — APPROVED** [review]
LGTM.

File: community/membership.md [hunk 2]
```diff
@@ -268,6 +268,7 @@ before being able to contribute effectively.
 
 A change in membership role may be approved by consensus of the Argo Project leads and approvers. By convention, role changes are discussed in a meeting with Argo Project approvers before being finalized. Role changes are generally reviewed on a quarterly basis.
 
+The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings at a reasonable size.
```
[the remark this answers is not in the document]
Comment (crenshaw-dev) [reply community/membership.md:271]: Should I just drop the last sentence?

[the remark this answers is not in the document]
Comment (hblixt) [reply community/membership.md:271]: IMO, it doesnt really add anything, so might as well.

**Comment by hblixt** [comment]
LGTM!

File: community/membership.md [hunk 3]
```diff
@@ -268,6 +268,7 @@ before being able to contribute effectively.
 
 A change in membership role may be approved by consensus of the Argo Project leads and approvers. By convention, role changes are discussed in a meeting with Argo Project approvers before being finalized. Role changes are generally reviewed on a quarterly basis.
 
+The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend.
```
Comment (jannfis) [inline community/membership.md:271]: Just a nit/suggestion to wording:

```suggestion
The membership meeting should be attended by project leads and by unscoped approvers (i.e. those who have access to merge _any_ PR in their subproject). Scoped approvers (i.e. those whose access is limited by the project's OWNERS file to a particular "scope" in the repo) do not attend.
```

**Comment by mayzhang2000** [comment]
+1

**Comment by edlee2121** [comment]
+1

**Comment by terrytangyuan** [comment]
+1

**Review by leoluz — APPROVED** [review]
lgtm

**Comment by zachaller** [comment]
+1

**Comment by jannfis** [comment]
+1

**Comment by ashutosh16** [comment]
+1

**Comment by crenshaw-dev** [comment]
+1

**Comment by ishitasequeira** [comment]
+1

**Comment by 34fathombelow** [comment]
+1

**Comment by jgwest** [comment]
+1

**Comment by wtam2018** [comment]
+1

**Review by anandf — COMMENTED** [review]
+1

**Comment by pasha-codefresh** [comment]
+1

**Comment by keithchong** [comment]
+1

**Comment by todaywasawesome** [comment]
+1

**Comment by kostis-codefresh** [comment]
+1

**Review by wanghong230 — APPROVED** [review]

**Comment by wanghong230** [comment]
+ 1 B

**Comment by reginapizza** [comment]
+1

**Comment by crenshaw-dev** [comment]
Thanks, all!

**Pull request merged**